lnbits/wallets/breez.py

```diff
@@ -0,0 +1,257 @@
+                return StatusResponse(f"Failed to connect to breez, got: '{exc}...'", 0)
+
+            return StatusResponse(None, int(node_info.channels_balance_msat))
+
+        async def create_invoice(
+            self,
+            amount: int,
+            memo: Optional[str] = None,
+            description_hash: Optional[bytes] = None,
+            unhashed_description: Optional[bytes] = None,
+            **kwargs,
+        ) -> InvoiceResponse:
+            # if description_hash or unhashed_description:
+            #     raise UnsupportedError("description_hash and unhashed_description")
+
+            if description_hash and not unhashed_description:
+                raise UnsupportedError(
+                    "'description_hash' unsupported by Greenlight, provide"
+                    " 'unhashed_description'"
+                )
+            breez_invoice: breez_sdk.ReceivePaymentResponse = (
+                self.sdk_services.receive_payment(
+                    breez_sdk.ReceivePaymentRequest(
+                        amount * 1000,  # breez uses msat
+                        unhashed_description.decode() if unhashed_description else memo,
+                        preimage=kwargs.get("preimage"),
+                        opening_fee_params=None,
+                        use_description_hash=True if unhashed_description else None,
+                    )
+                )
+            )
+
+            return InvoiceResponse(
+                True,
+                breez_invoice.ln_invoice.payment_hash,
+                breez_invoice.ln_invoice.bolt11,
+                None,
+            )
+
+        async def pay_invoice(
+            self, bolt11: str, fee_limit_msat: int
+        ) -> PaymentResponse:
+            invoice = lnbits_bolt11.decode(bolt11)
+
+            try:
+                send_payment_request = breez_sdk.SendPaymentRequest(bolt11=bolt11)
+                send_payment_response: breez_sdk.SendPaymentResponse = (
+                    self.sdk_services.send_payment(send_payment_request)
+                )
+                payment: breez_sdk.Payment = send_payment_response.payment
+            except Exception as exc:
+                logger.info(exc)
+                try:
+                    # try to report issue to Breez to improve LSP routing
+                    self.sdk_services.report_issue(
+                        breez_sdk.ReportIssueRequest.PAYMENT_FAILURE(
+                            breez_sdk.ReportPaymentFailureDetails(invoice.payment_hash)
+                        )
+                    )
+                except Exception as ex:
+                    logger.info(ex)
+                # assume that payment failed?
+                return PaymentResponse(
+                    False, None, None, None, f"payment failed: {exc}"
+                )
+
+            assert (
+                payment.status == breez_sdk.PaymentStatus.COMPLETE
+            ), "payment is pending"
```

Comment:
Do not raise, but return `PaymentResponse`.

```suggestion
            return PaymentResponse(False, None, None, None, "payment is pending" )
```